<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>kent fox</author>
 <class>QTalkWindow</class>
 <widget class="QMainWindow" name="QTalkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>808</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTalk</string>
  </property>
  <property name="locale">
   <locale language="Chinese" country="China"/>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="avatarButton">
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="searchEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid #aaa; border-radius:10px; background-color: palette(base);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="cursorPosition">
           <number>0</number>
          </property>
          <property name="placeholderText">
           <string> Search...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="friendsButton">
          <property name="styleSheet">
           <string notr="true">background:transparent; border-width:0px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="chatList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0px solid #aaa; border-radius:10px; background-color: palette(base);</string>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="resizeMode">
         <enum>QListView::Adjust</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QToolButton" name="chatIcon">
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="chatTitle">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="chatInfo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>14</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QToolButton" name="chatButton">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="msgBox">
        <property name="styleSheet">
         <string notr="true">border: 0px solid #aaa; border-radius:10px; background-color: palette(base);</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QToolButton" name="fileButton">
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputBox">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid #aaa; border-radius:10px; background-color: palette(base);</string>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Input your word...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="emojisButton">
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="sendButton">
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQTalk">
    <property name="title">
     <string>QTalk</string>
    </property>
    <widget class="QMenu" name="menuNotify">
     <property name="title">
      <string>Notify</string>
     </property>
     <addaction name="actionNotify1"/>
     <addaction name="actionNotify2"/>
     <addaction name="actionNotify3"/>
     <addaction name="actionNotify4"/>
     <addaction name="actionNotify5"/>
    </widget>
    <widget class="QMenu" name="menuThemes">
     <property name="title">
      <string>Themes</string>
     </property>
     <addaction name="actionThemeLight"/>
     <addaction name="actionThemeDark"/>
     <addaction name="separator"/>
     <addaction name="actionThemeAuto"/>
    </widget>
    <addaction name="menuNotify"/>
    <addaction name="menuThemes"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuQTalk"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNotify1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="actionNotify2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="actionNotify3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="actionNotify4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="actionNotify5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="actionThemeLight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionThemeDark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionThemeAuto">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>inputBox</tabstop>
  <tabstop>searchEdit</tabstop>
  <tabstop>sendButton</tabstop>
  <tabstop>chatList</tabstop>
  <tabstop>chatButton</tabstop>
  <tabstop>chatIcon</tabstop>
  <tabstop>avatarButton</tabstop>
  <tabstop>friendsButton</tabstop>
  <tabstop>emojisButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
